internal/cli/commands.go

```diff
@@ -36,8 +36,8 @@ func New() *cobra.Command {
 		SilenceUsage:      true,
 		PersistentPreRun: func(cmd *cobra.Command, args []string) {
 			http.DefaultTransport = userAgentTransport{http.DefaultTransport}
-			if cwd != "" {
-				if err := os.Chdir(cwd); err != nil {
+			if workDir != "" {
+				if err := os.Chdir(workDir); err != nil {
 					fmt.Printf("failed to change dir to %s: %v\n", cwd, err)
```

Comment:
```suggestion
					fmt.Printf("failed to change dir to %s: %v\n", workDir, err)
```

We should also drop `cwd` above? Or do something else with it? I'm not sure what the original intent was.

Reply:
Cwd is used as the default for workdir in the flag